### PR TITLE
Update submodule and fix broken items

### PR DESCRIPTION
**Pull request by laritakr**
# Story

After updating the Hyku submodule, there are some issues to resolve before merging. 
Refs https://github.com/scientist-softserv/palni_palci_knapsack/issues/164

# Expected Behavior Before Changes

# Expected Behavior After Changes

# Screenshots / Video

<details>
<summary></summary>

</details>

# Notes

### CODE REVIEW

**Review by kirkkwang — APPROVED** [review]
that looks good to me

File: spec/rails_helper.rb [hunk 1]
```diff
@@ -35,7 +35,7 @@
 
 RSpec.configure do |config|
   # Remove this line if you're not using ActiveRecord or ActiveRecord fixtures
-  config.fixture_path = "#{::Rails.root}/spec/fixtures"
+  config.fixture_path = Rails.root.join("spec", "fixtures").to_s
```
Comment (sjproctor) [inline spec/rails_helper.rb:38]: You might want single quotes here since you removed the interpolation

**Review by sjproctor — APPROVED** [review]
One non-blocking comment. Looks good!

File: spec/rails_helper.rb (hunk 1 above)
Comment (laritakr) [reply spec/rails_helper.rb]: Odd... that was a rubocop auto-fix! I wouldn't have expected it to use double quotes.

**Pull request merged**